src/realmGeolContinental.ttl

```diff
@@ -127,12 +132,18 @@ soreagcont:Craton rdf:type owl:Class ;
                               owl:onProperty sorelph:hasPlanetaryStructure ;
                               owl:allValuesFrom soreagcont:Shield
                             ] ;
-            rdfs:label "craton"@en .
+            dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+            dcterms:source <http://www.glossaryofgeology.org> ;
+            skos:definition "A craton is part of Earth's continental crust that has attained stability and has been little deformed for a prolonged period. Cratons include shield areas, where Precambrian rocks are exposed, and platform areas, where Precambrian rocks are overlain by a thin layer of Phanerozoic strata. (Neuendorf et al., 2011)"@en ;                            
```

Comment:
"little deformed over a prolonged period" is not an ideal part of a definition for craton as it is ambiguous and subjective, not to mention that many cratons are highly deformed at their margins. 
Also a craton includes all of the lithosphere, not just the crust. Age, stability, thickness are some of the key criteria here.
Suggested def: A relatively old, stable, and thick part of the Earth's lithosphere that forms the nucleus of a continent.

Reply:
restrict to continental lithosphere?
A relatively old, stable, and thick part of the Earth's continental lithosphere that forms the nucleus of a continent.

sort of implicit in 'nucleus of continent' I guess, avoids possible interpretation it could be oceanic lithosphere forming nucleus of continent?

Reply:
I think so. The thickness also implies continental, but there is ambiguity in just 'relatively thick lithosphere' which could be continental lithosphere, cratons, or oceanic plateaux. So the restriction is good. 

I now wonder if this def implies that all continents have cratonic nucleii (do they??)